bin/logger.ts

```diff
@@ -0,0 +1,45 @@
+type LoggerLevel = 'debug' | 'info' | 'warn' | 'error' | 'silent';
```

Comment:
Hey @carlosallexandre do you think it's worth using https://www.npmjs.com/package/pino for a logging lib? Pretty popular, saves us making our own?

Reply:
Good question, maybe add into this PR for now, unless there is lots to do, then another PR would be good then remove logging in this one, if that makes sense?

Reply:
Sure...
I will work on this right now.